core/src/main/java/edu/wpi/grip/core/settings/ProjectSettings.java

```diff
@@ -12,21 +12,48 @@
  */
 public class ProjectSettings implements Cloneable {
 
-    @Setting(label = "FRC Team Number", description = "The team number, if used for FRC")
+    @Setting(label = "FRC team number", description = "The team number, if used for FRC")
     private int teamNumber = 0;
 
-    @Setting(label = "NetworkTables Server Address", description = "The host that runs the Network Protocol server. " +
-            "If not specified and NetworkTables is specified as the protocol, the hostname is derived from the team " +
+    @Setting(label = "NetworkTables server address", description = "The host that runs the Network " +
+            "Protocol server.  If not specified and NetworkTables is used, the hostname is derived from the team " +
             "number.")
-    private String publishAddress = "";
+    private String publishAddress = computeFRCAddress(teamNumber);
 
-    @Setting(label = "Deploy Address", description = "The remote host that grip should be remotely deployed to. If " +
-            "not specified, the hostname is derived from the team number.")
-    private String deployAddress = "";
+    @Setting(label = "Default deploy address", description = "The remote host that grip should be deployed to. " +
+            "If not specified, the hostname is derived from the team number.")
+    private String deployAddress = computeFRCAddress(teamNumber);
 
+    @Setting(label = "Default deploy directory", description = "The directory on the remote host to deploy GRIP to.")
+    private String deployDir = "/home/lvuser";
+
+    @Setting(label = "Default deploy user", description = "The username to log in with when deploying over SSH.")
+    private String deployUser = "lvuser";
+
+    @Setting(label = "Default deploy java home", description = "Where Java is installed on the robot.")
+    private String deployJavaHome = "/usr/local/frc/JRE/";
```

Comment:
I don't like that this is specific to FRC by default.
I think we should do this differently, perhaps a button to assign the FRC defaults.